odoo/addons/base/models/ir_actions.py

```diff
@@ -1170,57 +1170,63 @@ def run(self):
         """
         res = False
         for action in self.sudo():
-            action_groups = action.group_ids
```

Comment:
@odoo/rd-framework-py Hi :) 

We didn't have the time to ask a review for it (we had 3 days to merge it, and a lot of more important blocking stuff to unblock), but you might be interested by that change. It's just move of code, but for more context: https://discord.com/channels/678381219515465750/844122378065412127/1400871365300588644

The post merge change continue there: https://github.com/odoo/enterprise/pull/91601